cloud/pkg/admissioncontroller/admission.go

```diff
@@ -84,7 +99,19 @@ func Run(opt *options.AdmissionOptions) error {
 	controller.Client = cli
 	controller.CrdClient = vcli
 
-	caBundle, err := os.ReadFile(opt.CaCertFile)
+	// generate ca certs
+	if err := genCerts(opt); err != nil {
```

Comment:
when admission have multi replicas, there has some problems, the latter replicas will gen certs again and will cover the former replicas, so two replicas has different cabunlde and kube-apiserver will has error when access the admisson

Reply:
yes, so store the ca and certs in secret, and check whether secret exsit or not, if exist, read from it, or generate it, 
just like what shell script `build/admission/gen-admission-secret.sh` did